.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
Editor config files really shouldn't be in the repo (unless the project has a standard editor, and this little mini module does not).